docs/en/solutions/How_to_Use_Pipeline_to_Train_AI_Models.md

```diff
@@ -0,0 +1,952 @@
+                if [ -z "$pod_status" ]; then
+                  continue
+                fi
+                if [ "$pod_status" = "Pending" ]; then
+                  echo "Pod ${pod_name} is pending, waiting..."
+                  sleep 1
+                  continue
+                fi
+                if [ "$pod_status" != "Running" ]; then
+                  break
+                fi
+                sleep 1
+              done
+
+              echo ""
+              echo "--- Pod ${pod_name} logs completed, status: ${pod_status} ---"
+              if [ "$pod_status" != "Succeeded" ]; then
+                exit 1
+              fi
+            securityContext:
+              runAsNonRoot: true
+      timeout: 720h0m0s
+  finally: []
+```
+
+</details>
+
+### Pipeline Parameters
+
+The pipeline includes the following key parameters that need to be configured:
+
+**Repository Parameters:**
+- `MODEL_REPO_URL`: Git URL of the model repository
+- `MODEL_REPO_BRANCH`: Branch of the model repository (optional)
+- `DATASET_REPO_URL`: Git URL of the dataset repository
+- `DATASET_REPO_BRANCH`: Branch of the dataset repository (optional)
+- `OUTPUT_MODEL_REPO_URL`: Git URL of the output model repository
+- `OUTPUT_MODEL_REPO_BRANCH`: Branch of the output model repository (optional, auto-generated if not specified)
+- `GIT_CREDENTIAL_SECRET_NAME`: Secret name of the git credential secret (requires `GIT_USER` and `GIT_TOKEN` keys, where `GIT_USER` stores the git username and `GIT_TOKEN` stores the git password or access token)
+
+**Training Parameters:**
+- `TRAINING_IMAGE`: Container image for the training job
+- `TRAIN_ARG_IMAGE_SIZE`: Training argument image size (default: "640")
+- `TRAIN_ARG_BATCH_SIZE`: Training argument batch size (default: "16")
+- `TRAIN_ARG_EPOCHS`: Training argument epochs (default: "3")
+- `TRAIN_ARG_DATA`: Training argument data file (default: "coco128.yaml")
+- `TRAIN_ARG_WEIGHTS`: Training argument weights (default: "models/yolov5n.pt")
+- `TRAIN_ARG_WORKER`: Training argument worker (default: "0")
+- `TRAIN_ARG_DEVICE`: Training device, multiple devices separated by comma (default: "0")
+
+**Resource Parameters:**
+- `STORAGE_SIZE`: Storage size (default: "5Gi")
+- `STORAGE_CLASS`: Storage class name
+- `REPLICAS`: Number of replicas (default: "1", distributed training will be enabled if greater than 1)
+- `CPU_REQUEST`: Request CPU (default: "1", leave empty to not request CPU)
+- `MEMORY_REQUEST`: Request memory (default: "8Gi", leave empty to not request memory)
+- `CPU_LIMIT`: Limit CPU (default: "8", leave empty to not limit CPU)
+- `MEMORY_LIMIT`: Limit memory (default: "20Gi", leave empty to not limit memory)
+- `NVIDIA_GPUALLOC`: NVIDIA GPU allocation - number of GPU cards (default: "1", leave empty to not allocate GPU)
+- `NVIDIA_GPUCORES`: NVIDIA GPU cores - percentage of compute power per card, range 1-100 (default: "50", leave empty to not configure GPU cores)
```

Comment:
This is only available when using `HAMi`, maybe should let user to choose what kind of GPU device plugin to use?